test/rules/experiments/nnbd/rules/unnecessary_null_checks.dart

```diff
@@ -54,3 +54,14 @@ f5(int? p) {
   int? v1;
   v1 ??= p!; // OK
 }
+
+class B {
+  m3a(int? p) => p!; // OK
+  m3b(int? p) {
+    return p!; // OK
+  }
+}
+
+f6(int? p) {
```

Comment:
Why do we put `f6` not after `f5`?
I may guess that we want to group unresolved / error cases at hte end?

Reply:
No good reason.  It was just the order I added them. 😄 

Re-sorted.